README.md

```diff
@@ -151,17 +172,106 @@ tailscale up --advertise-routes="10.0.0.0/24,192.168.0.0/24"
 ```
 
 ### Can I route traffic from machines on my local network to Tailscale endpoints automatically?
-In theory, yes - however it does require manual changes to your routing rules and these will need
-to be updated if you take advantage of WAN fail-over. This has been discussed in more detail
-[here](https://github.com/SierraSoftworks/tailscale-udm/discussions/51).
 
-*Note that we do not currently include this in `tailscale-udm` due to the risk of breaking conflicts in future.*
+Yes! As of January 30, 2025, [two changes][1][2] to Tailscale have made this
+possible. Much credit goes to @tomvoss, who contributed significant effort to
+the initial implementation, detailed [here][tailnet-routing-discussion].
+Before going further please read tailscale's [subnet router
+documentation][tailscale-subnet-router-docs] and familiarize yourself with the
+concepts of subnet routers, independent of Unifi OS.
+
+#### Prerequisites
+
+Before proceeding, please review Tailscale’s [subnet router
+documentation][tailscale-subnet-router-docs] to understand the core concepts of
+subnet routing, independent of UniFi OS.
+
+Note: You do not need to manually enable `net.ipv4.ip_forward` on your UniFi OS
+device—it is enabled by default. If you want to confirm its status, run:
+
+```sh
+sysctl net.ipv4.ip_forward
+```
+
+#### Installation and Configuration
+
+If Tailscale is already installed, update its configuration:
+
+1. Remove the `--tun userspace-networking` flag from `/etc/default/tailscaled`.
+
+2. Add the following instead:
+
+   ```text
+   --socket=/var/run/tailscale/tailscaled.sock
+   ```
+
+   `FLAGS` in `/etc/default/tailscaled` should look something like this:
+
+   ```text
+   FLAGS="--state=/data/tailscale/tailscaled.state --socket=/var/run/tailscale/tailscaled.sock"
+   ```
+
+3. Restart the Tailscale daemon:
+
+   ```sh
+   systemctl restart tailscaled
+   ```
+
+If Tailscale is not yet installed, first export the required environment variable:
+
+```
+export TAILSCALED_FLAGS="--socket=/var/run/tailscale/tailscaled.sock"
+```
+
+Then, proceed with the [installation steps](#installation).
```

Comment:
I'm open to us publishing a v3.x of this package which moves away from userspace networking by default (potentially having a flag to install it with userspace mode enabled). Any concerns about us doing so (and subsequently removing this documentation as it will no longer be necessary)?

And to be clear, I'm not asking you to undertake that work or suggesting that we block publishing of this guidance until it is done.

Reply:
100% that is my goal. I just wanted to iterate towards that rather than making the jump because of the unknown unknowns.

Reply:
@notheotherben agree. https://github.com/SierraSoftworks/tailscale-udm/pull/106 MUST be merged before these docs are though so people are led down a bug path.